app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -112,7 +111,7 @@ export class SearchAndAddOrder extends React.Component {
             tabName="OutPatient">
             {this.renderSearchDrug()}
             {this.renderAddForm(this.props.outpatientCareSetting)}
-            {(this.state.isDelete || this.props.draftOrders.length > 0) &&
+            {(this.state.isDelete && this.props.draftOrders.length > 0) &&
```

Comment:
Why did you use `&&` is `this.state.isDelete` always true?

Reply:
Oversight, will have that corrected

Reply:
Yeah oversight on testing implementation, would fix this now